crates/telio-pq/src/conn.rs

```diff
@@ -79,11 +82,13 @@ impl ConnKeyRotation {
                         #[allow(mpsc_blocking_send)]
                         let _ = chan.send(super::Event::Rekey(wg_keys)).await;
                     }
-                    Err(err) => {
+                    Ok(Err(err)) => {
```

Comment:
I didn't look into the call site, maybe it makes sense, but can't we get rid of double indirection? If `Ok()` then it should be a legit value.

Reply:
The outer `Result<>` is imposed by `tokio::time::timeout`. I don't see a way to change it

Reply:
TBH, it does represent the actual situation kind of nicely IMHO.

Because we have two results, one coming from task execution (e.g. OK means executed Err means some execution error). If we get OK, which means, that task executed till its end -> the task itself may have failed for some inner reason.

But IMHO it would be nice to print out the value of `_timeout`  on line 90. While it probably will ever print `Elapsed`, it is still kind of nice confirmation :D

Reply:
`_timeout` is of type `Elapsed` which is empty (doesn't contain the duration). But I can print the duration because I have access to it anyway